packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -304,7 +304,7 @@ export class ListItem
 
   @Listen("calciteInternalListItemGroupDefaultSlotChange")
   @Listen("calciteInternalListDefaultSlotChange")
-  handleCalciteInternalListDefaultSlotChanges(event: CustomEvent<void>): void {
+  handleCalciteInternalListDefaultSlotChanges(event: CustomEvent): void {
```

Comment:
Do Lumina's type set `CustomEvent.detail`'s' type as `void` by default?

Reply:
yes!

to be more precise:
- default payload type is `undefined`
- it is a typescript error to not provide a payload to the `.emit()` function if the event has non-`undefined` payload type
- the codemod will update `EventEmitter<void>` to `EventEmitter` as needed

Reply:
edit:
Regarding listening to events, `CustomEvent.detail` is still `any` since that's coming from TypeScript.
I also see that you used `CustomEvent<void>` in many places, so I will revert this place to use `CustomEvent<void>` for consistency